### ISSUE TITLE
Add option to specify user / password

### ISSUE THREAD

**Issue by vzhomeexperiments** [jwijffels/taskscheduleR#83]
Hi, I am currently exploring this package and have a question:

is it possible to add support of username and password when creating a task?

This seems to be possible but there is no such parameters in the function:

`schtasks /Create
[/S system [/U username [/P [password]]]]
[/RU username [/RP [password]] /SC schedule [/MO modifier] [/D day]
[/M months] [/I idletime] /TN taskname /TR taskrun [/ST starttime]
[/RI interval] [ {/ET endtime | /DU duration} [/K]
[/XML xmlfile] [/V1]] [/SD startdate] [/ED enddate] [/IT] [/Z] [/F]`

basically once the task is running the black window would popup which is kind of annoying 

Thank you

**Comment by jwijffels** [comment]
you can extra elements in argument `schtasks_extra ` of `taskscheduler_create ` as in `taskscheduler_create (..., schtasks_extra  = "/RP supersecret")`

**Comment by vzhomeexperiments** [comment]
thanks, just found it: `schtasks_extra = "/RU superuser /RP supersecret"` will test that now
